### ISSUE TITLE
[quarkus-maven-plugin] No way to specify ',' in -Dquarkus.native.additional-build-args

### ISSUE THREAD

**Issue by jerboaa** [quarkusio/quarkus#14582]
**Describe the bug**
For some reason `-Dquarkus.native.additional-build-args="-J-Xdebug -J-Xrunjdwp:transport=dt_socket,server=y,address=88888"` turns into something like this via NativeBuildStep:

```
native-image -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=1 -J-Duser.language=en -J-Dfile.encoding=UTF-8 -J-Xdebug -J-Xrunjdwp:transport=dt_socket server=y address=88888 --initialize-at-build-time= -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy\$BySpaceAndTime -H:+JNI -H:+AllowFoldMethods -jar quarkus-integration-test-bouncycastle-1.11.0.Final-runner.jar -H:FallbackThreshold=0 -H:+ReportExceptionStackTraces -H:-AddAllCharsets -H:EnableURLProtocols=http -H:NativeLinkerOption=-no-pie -H:-UseServiceLoaderFeature -H:+StackTrace quarkus-integration-test-bouncycastle-1.11.0.Final-runner
```

Note the `-J-Xrunjdwp:transport=dt_socket server=y address=88888` part in the native-image invocation.

**Expected behavior**
Commas passed through properly.

**Actual behavior**
Commas get turned into spaces?

**To Reproduce**

```
$ mvn verify -f integration-tests/pom.xml -pl 'bouncycastle' -Dnative -Dquarkus.native.additional-build-args="-J-Xdebug -J-Xrunjdwp:transport=dt_socket,server=y,address=88888"
```

**Additional context**
This is quarkus 1.11.0.Final. The maven command fails with something like:

```
[INFO] --- quarkus-maven-plugin:1.11.0.Final:native-image (native-image) @ quarkus-integration-test-bouncycastle ---
[INFO] [org.jboss.threads] JBoss Threads version 3.2.0.Final
[INFO] [io.quarkus.deployment.pkg.steps.JarResultBuildStep] Building native image source jar: /home/sgehwolf/Documents/openjdk/quarkus/quarkus-source/integration-tests/bouncycastle/target/quarkus-integration-test-bouncycastle-1.11.0.Final-native-image-source-jar/quarkus-integration-test-bouncycastle-1.11.0.Final-runner.jar
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Building native image from /home/sgehwolf/Documents/openjdk/quarkus/quarkus-source/integration-tests/bouncycastle/target/quarkus-integration-test-bouncycastle-1.11.0.Final-native-image-source-jar/quarkus-integration-test-bouncycastle-1.11.0.Final-runner.jar
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Running Quarkus native-image plugin on Snapshot
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] /disk/graal/upstream-sources/graal/mandrel-build/bin/native-image -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=1 -J-Duser.language=en -J-Dfile.encoding=UTF-8 -J-Xdebug -J-Xrunjdwp:transport=dt_socket server=y address=88888 --initialize-at-build-time= -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy\$BySpaceAndTime -H:+JNI -H:+AllowFoldMethods -jar quarkus-integration-test-bouncycastle-1.11.0.Final-runner.jar -H:FallbackThreshold=0 -H:+ReportExceptionStackTraces -H:-AddAllCharsets -H:EnableURLProtocols=http -H:NativeLinkerOption=-no-pie -H:-UseServiceLoaderFeature -H:+StackTrace quarkus-integration-test-bouncycastle-1.11.0.Final-runner
Error: Unknown arguments: address=88888, quarkus-integration-test-bouncycastle-1.11.0.Final-runner
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  15.875 s
[INFO] Finished at: 2021-01-25T15:58:43+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.11.0.Final:native-image (native-image) on project quarkus-integration-test-bouncycastle: Failed to generate native image: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: java.lang.RuntimeException: Failed to build native image
[ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:307)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:972)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:834)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] Caused by: java.lang.RuntimeException: Image generation failed. Exit code: 1
[ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:572)
[ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:273)
[ERROR] 	... 10 more
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by jerboaa** [comment]
OK this is documented in:
https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/writing-native-applications-tips.adoc#delaying-class-initialization (second "Note").

Then you'd have to know to take this to your advantage by using `,` as separator for multiple arguments.

TLDR; The fix is: `-Dquarkus.native.additional-build-args="-J-Xdebug,-J-Xrunjdwp:transport=dt_socket\\,server=y\\,address=8888"

Thanks to @zakkak!
